homeassistant/components/uptime_kuma/binary_sensor.py

```diff
@@ -0,0 +1,38 @@
+"""Summary binary data from Uptime Kuma."""
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Defer sensor setup to the shared sensor module."""
+    coordinator = hass.data[DOMAIN]
+    # print(coordinator.data)
+    async_add_entities(
+        UptimeKumaBinarySensor(coordinator, monitor) for monitor in coordinator.data
+    )
+
+
+class UptimeKumaBinarySensor(BinarySensorEntity, CoordinatorEntity):
+    """Represents an Uptime Kuma binary sensor."""
+
+    def __init__(self, coordinator, monitor):
+        """Initialize the Uptime Kuma binary sensor."""
+        super().__init__(coordinator)
+        self._name = monitor
```

Comment:
Could you use `EntityDescription`?
Or at least use `_attr_` for the name and the icon?